admin/src/js/controllers/logout-confirm.js

```diff
@@ -0,0 +1,20 @@
+angular.module('controllers').controller('LogoutConfirmCtrl',
+  function(
+    $scope,
+    $uibModalInstance,
+    Session
+  ) {
+    'use strict';
+    'ngInject';
+
+    $scope.submit = function() {
+      $scope.setProcessing();
+      Session.logout();
+      $uibModalInstance.close();
+    };
+
+    $scope.cancel = function() {
+      $uibModalInstance.dismiss();
+    };
+  }
+);
```

Comment:
I think we can remove the confirm dialog. We use that in the main app because we don't want phone users to log out accidentally and then have to log in again. Admins are less likely to do that by accident because they're on a desktop, and also it's easy for them to log in again.